test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Admin/DatabaseAdminTests.cs

```diff
@@ -85,5 +85,25 @@ private LiveConnectionHelper.TestConnectionResult GetLiveAutoCompleteTestObjects
             requestContext.VerifyAll();
         }
 
+        /// <summmary>
+        /// Get database info test
+        /// </summary>
+        // [Fact]
```

Comment:
I followed the tests previously in the file, didn't realize commenting out the Fact attribute comments out the test.